app/swagger/requests/gibct/institutions.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+module Swagger
+  module Requests
+    module Gibct
+      class Institutions
+        include Swagger::Blocks
+
+        swagger_path '/v0/gi/institutions/autocomplete' do
+          operation :get do
+            key :description, 'Retrieves all institution names begining with a set of letters'
```

Comment:
It won't return _all_ institution names - we put a limit on it for performance. But I don't think we should document the max per request in the API docs since it's more of an implementation detail that we might choose to tune.